certificate-service/integration-test/src/test/java/se/inera/intyg/certificateservice/integrationtest/fk7211/FK7211ActiveIT.java

```diff
@@ -2737,6 +2749,27 @@ void shallReturn403IfVersionNotValid() {
 
       assertEquals(400, response.getStatusCode().value());
     }
+
+    @Test
+    @DisplayName("FK7211 - Om intyget signeras ska ett meddelande läggas på AMQn")
+    void shallSuccessfullyAddMessageOfSigningOnAMQ() {
+      final var testCertificates = testabilityApi.addCertificates(
+          customTestabilityCertificateRequest(FK7211, VERSION)
+              .unit(ALFA_MEDICINCENTRUM_DTO)
+              .build()
+      );
+
+      api.signCertificate(
+          customSignCertificateRequest()
+              .unit(ALFA_MEDICINCENTRUM_DTO)
+              .build(),
+          certificateId(testCertificates),
+          version(testCertificates)
+      );
+
+      waitAtMost(Duration.ofSeconds(30))
+          .untilAsserted(() -> assertFalse(testListener.messages.isEmpty()));
```

Comment:
Would be great if we could assert the message. It is a simple message containing the certificateId and type. Would be great to verify that it is a correct message.

Reply:
Done